src/app/(proper_react)/(redesign)/(authenticated)/user/welcome/EnterInfo.tsx

```diff
@@ -46,7 +50,9 @@ const createProfileAndStartScan = async (
 
   const result: WelcomeScanBody = await response.json();
   if (!result?.success) {
-    throw new Error("Could not start scan");
+    logger.error("Could not start scan", { response });
+    res?.status(403).json({ success: false });
+    return { success: false };
```

Comment:
Here you could `return redirect("/")` (or wherever is appropriate). `/` is probably good because it will return to the landing page if there is an authentication problem, or automatically redirect to the dashboard if not.

The only thing to consider is if it's possible to cause a redirect loop, I don't think it should since this user is ineligible and should not have been able to make it here in the first place.

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4725/commits/20fb12b952fc94b9869e8d038f49bfb4a636405e